parser.js

```diff
@@ -32,8 +33,11 @@ exports.parseForESLint = function parseForESLint(code, options) {
             options = Object.assign({}, options, { jsx: tsx });
         }
     }
+    if (typeof options.generateServices === "boolean" && options.generateServices) {
+        generateServices = true;
+    }
 
-    const ast = parse(code, options);
+    const { ast, services } = generateServices ? converter.parseAndGenerateServices(code, options) : { ast: converter.parse(code, options) };
```

Comment:
I wonder if putting this behind a flag is really necessary. I do think it's a breaking change (in case any existing rules are checking for parser services), but it seems reasonable to me to release this in a major version. Please let me know if I'm missing something, though!

This project has been iterating pretty fast (which is a really welcome change!), and I don't think it's a terrible thing to release a few major versions quickly as we get this up to speed.

Reply:
I'm happy to do it that way. @platinumazure, any thoughts on this aspect?

Reply:
No concerns here! If we wanted to release this as semver-minor then I would want the flag, but I see no issue with doing a major release soon.